### PR TITLE
Resolve model Rss extends wrong class

### PR DESCRIPTION
**Pull request by edenduong**
### Description (*)

Please see: app/code/Magento/Review/Model/Rss.php

image

- The `__construct()` is override the parent class, but it doesn't call the `parent::__construct()`.

### Manual testing scenarios (*)

**Unit test covered.**

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Review/Model/Rss.php [hunk 1]
```diff
@@ -6,10 +6,11 @@
...
 
 /**
- * Class Rss
- * @package Magento\Catalog\Model\Rss\Product
+ * Rss Model
+ *
+ * Class \Magento\Catalog\Model\Rss\Product\Rss
  */
-class Rss extends \Magento\Framework\Model\AbstractModel
+class Rss
```
Comment (sidolov) [inline app/code/Magento/Review/Model/Rss.php]: Unfortunately, we can't remove inheritance according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/). Someone can use Rss model in 3d party code and use methods from the `\Magento\Framework\Model\AbstractModel`
It's better to call the parent constructor here, looks like that call was missed during the refactoring.

**Review by sidolov — CHANGES_REQUESTED** [review]

File: app/code/Magento/Review/Model/Rss.php (hunk 1 above)
Comment (edenduong) [reply app/code/Magento/Review/Model/Rss.php]: @sidolov: I have changed it. Now, I think it following the Backward Compatible Development Guide. 
Thank you very much.

**Review by sidolov — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-5671](https://jira.corp.magento.com/browse/ENGCOM-5671) has been created to process this Pull Request
:eight_spoked_asterisk: @sidolov, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

**Pull request merged**